services/bots/src/github-webhook/github-webhook.service.ts

```diff
@@ -13,8 +13,33 @@ export class GithubWebhookService {
   }
 
   async handleWebhook(params: WebhookHandlerParams): Promise<void> {
+    ISSUE_UPDATES[params.deliveryId] = {
+      owner: params.payload.repository?.full_name.split('/')[0],
+      repo: params.payload.repository?.full_name.split('/')[1],
+      issue_number: params.payload.number,
+      labels: [],
+      comments: [],
+    };
+
     await Promise.all(WEBHOOK_HANDLERS.map((handler) => handler.handle(params)));
-  }
 
-  async test() {}
+    if (ISSUE_UPDATES[params.deliveryId].labels.length) {
+      await this.githubApiClient.issues.addLabels({ ...ISSUE_UPDATES[params.deliveryId] });
+    }
+    if (ISSUE_UPDATES[params.deliveryId].comments.length) {
+      await this.githubApiClient.issues.createComment({
+        ...ISSUE_UPDATES[params.deliveryId],
+        body: ISSUE_UPDATES[params.deliveryId].comments
+          .map(
+            (entry) =>
+              `${entry.context}${
+                ISSUE_UPDATES[params.deliveryId].comments.length >= 2
+                  ? `\n<sub><sup>(message by ${entry.context})</sup></sub>`
+                  : ''
+              }`,
+          )
+          .join('\n\n---\n\n'),
+      });
+    }
+  }
```

Comment:
It should clean up the context. Preferably we do this in a `try…finally` so it's guaranteed to be cleaned up